### DISCUSSION TITLE
Chords that span staves

### DISCUSSION THREAD

**Discussion by williamclocksin** [w3c/mnx#310] [Ideas]
Being new to mnx, I cannot see a way to represent a chord in which some of its noteheads are on one staff and some of its noteheads are on another staff. In Calliope (a music notation system from the 1990s) I represented noteheads as having not only a step etc but also a staff. Most of the time this staff is the same as the note event is on, but in some cases it isn't. It was also useful when dragging big chords around the system, because the staff field of a notehead would be set to null before it found a staff to attach to.

**Comment by adrianholovaty** [comment]
Yes, we already have a solution to this — it's the `"staff"` key on the [note object](https://w3c.github.io/mnx/docs/mnx-reference/objects/note/). That's pretty buried in the documentation, though, and we don't yet have an example document...!
